core/src/main/java/edu/ucr/cs/riple/core/metadata/index/Fix.java

```diff
@@ -61,7 +62,7 @@ public static Factory<Fix> factory(FieldDeclarationAnalysis analysis) {
               field.variables.addAll(variables);
             });
       }
-      return new Fix(new Change(location, infos[7], true), infos[6], infos[8], infos[9]);
+      return new Fix(new Change(location, config.nullableAnnot, true), info[6], info[8], info[9]);
```

Comment:
FYI, the tests on CI are going to fail since it is using the previous version of `NullAway`, and the precondition above it causing the termination with error.

Reply:
@lazaroclapp For unknown reason, `github` does nothing when I re-request a review from you, please find the changes applied to your comment.

Reply:
> FYI, the tests on CI are going to fail since it is using the maven central version of NullAway, and the precondition above is causing the termination with error.

Um, I imagine you want to make other changes to the auto-annotator, so probably we don't want to delay landing this. If we had more than one user for this project, the right call would probably to version our TSV formats and keep support for both versions for a while, but I guess that's overkill here. At the same time, I don't think the preconditions check is worth dealing with a broken CI...

I think my suggestion here would be to comment out the preconditions check with a TODO+Issue to re-enable it once NullAway 0.9.9 has been released. It's a bit hacky, but preferable to all the alternatives above. Apologies for the bikeshedding :)